packages/main/src/components/Page/index.tsx

```diff
@@ -14,21 +14,52 @@ import { CommonProps } from '../../interfaces/CommonProps';
 import styles from './Page.jss';
 
 export interface PagePropTypes extends CommonProps {
+  /**
+   * The title text appearing in the page header bar.
+   */
   title?: string;
+  /**
+   * It is used to set the background color of a page.
+   */
   backgroundDesign?: PageBackgroundDesign;
+  /**
+   * The (optional) custom header of this page. Use this aggregation only when a custom header
+   * is constructed where the default header consisting of title text + nav button is not sufficient.
+   */
   customHeader?: ReactNode;
+  /**
+   * The (optional) custom footer of this page.
+   */
   customFooter?: ReactNode;
+  /**
+   * A back button will be rendered on the left area of header bar if this property is set to true.
+   */
   showBackButton?: boolean;
+  /**
+   * Whether this page shall have a footer.
+   */
   showFooter?: boolean;
+  /**
+   * Whether this page shall have a header.
+   */
   showHeader?: boolean;
+  /**
+   * This event is fired when Nav Button is pressed.
+   */
   onNavButtonPress?: (e: CustomEvent<{}>) => void;
+  /**
+   * The page has three distinct areas that can hold content - a header, content area and a footer.
```

Comment:
```suggestion
   * The content area of the page.
```
footer and header are separate props, so content area is the only which is left here